js/createPost.js

```diff
@@ -0,0 +1,60 @@
+import { API_BASE_URL } from "./const.mjs";
+const token = localStorage.getItem("accessToken");
+
+/**
+ * Create a new post.
+ *
+ * @param {string} url - The URL to create a new post.
+ * @param {Object} newPost - The new post data, including title, body, and media.
+ */
+async function createPost(url, newPost) {
+  try {
+    const postData = {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${token}`,
+      },
+      body: JSON.stringify(newPost),
+    };
+    const response = await fetch(url, postData);
+    console.log(response);
+
+    if (response.ok) {
+      const json = await response.json();
+      // console.log(json);
+
+      document.getElementById("postTitle").value = "";
+      document.getElementById("postBodyArea").value = "";
+      document.getElementById("postMedia").value = "";
+
```

Comment:
Here, you can inform the user about successful post creation by sending a message via `alert()`